tests/utils.py

```diff
@@ -13,6 +16,22 @@
 from tests.mockserver import MockServer
 
 
+def create_scrapy_settings(request):
+    """Default scrapy-poet settings"""
+    s = dict(
+        # collect scraped items to crawler.spider.collected_items
+        ITEM_PIPELINES={
+            "tests.utils.CollectorPipeline": 100,
```

Comment:
Note that you can use (since Scrapy 2.4, and we require 2.6) type objects instead of full names here.